### DISCUSSION TITLE
How to join this organization?

### DISCUSSION THREAD

**Discussion by Infinity-chakra** [Your-First-Open-Source-Project/Your-First-Open-Source-Project#529] [Q&A] [answered]
I want to joint this community and contribute with my open source projects.
But it doesn't go well, first of all, I can't find the way to join, cuz here isn't join text highlighted with link.
Please help me.

**Comment by YSHgroup** [comment]
Hi @Infinity-chakra ,
You have to find the text [Start Here](https://github.com/Your-First-Open-Source-Project/start-here) under the title [Check out the following repositories below ](https://github.com/Your-First-Open-Source-Project#check-out-the-following-repositories-below-) in readme.md
If then, you can find the subtitle [How to join this organization](https://github.com/Your-First-Open-Source-Project/start-here#how-to-join-this-organization) in Readme there

**Reply by Infinity-chakra** [reply, marked as answer]
@YSHgroup , thank you, I'm in this org now.
